lib/compat/wordpress-6.4/block-hooks.php

```diff
@@ -79,6 +79,11 @@ function gutenberg_add_hooked_blocks( $settings, $metadata ) {
 
 	$exposed_settings = array_intersect_key( $settings, $fields_to_pick );
 
+	// If the block has a variations callback, call it and add the variations to the block.
```

Comment:
I think we will want to add any compat shims to the wordpress-6.5 folder instead of this one.

Reply:
They way it is loaded, I had to remove the function entirely from 6.4 and add it to 6.5.